resource-managers/yarn/src/main/scala/org/apache/spark/scheduler/cluster/YarnClientSchedulerBackend.scala

```diff
@@ -111,7 +111,7 @@ private[spark] class YarnClientSchedulerBackend(
     override def run() {
       try {
         val YarnAppReport(_, state, diags) =
-          client.monitorApplication(appId.get, logApplicationReport = true)
+          client.monitorApplication(appId.get, logApplicationReport = false)
```

Comment:
Yes, it's too verbose currently in the client mode. I remembered we only have such output in cluster mode YARN client. My only concern is that turning to `false` will also lose the detailed reports. I think it would be better if we still have the detailed report when state is changed.